forge-gui/res/cardsfolder/upcoming/gixs_caress.txt

```diff
@@ -0,0 +1,7 @@
+Name:Gix's Caress
+ManaCost:2 B
+Types:Sorcery
+A:SP$ Discard | ValidTgts$ Opponent | NumCards$ 1 | DiscardValid$ Card.nonLand | Mode$ RevealYouChoose | SubAbility$ DBToken| SpellDescription$ Target opponent reveals their hand. You choose a nonland card from it. That player discards that card.\nCreate a tapped Powerstone token. (It's an artifact with "{T}: Add {C}. This mana can’t be spent to cast a nonartifact spell.")
```

Comment:
- [ ] spacing
- [ ] Break up SpellDescription (token stackdescription feeds off SpellDescription) and get rid of `\n`
- [ ] fix `’` in this line and Oracle